app/interface/basiccli.py

```diff
@@ -33,13 +34,43 @@ def start(self):
             raise RuntimeError("Matcher has not been set.")
 
         while True:
-            question = input("Please enter your question >> ")
-            print("Loading....")
+
+            year = questionary.select(
+                "What year do you want to search?",
+                choices=["2017", "2018", "2019"],
+            ).ask()
```

Comment:
With the current design of injecting the question matcher when the interface is created, it would not be possible to select the year of interest from within the interface. Instead, imo, it would make sense as an argument when launching the app. `python main.py --year 2019` although after writing that it feels clunky. Point for discussion at next meeting?

Other than that I think PR looks great and happy to approve.

Reply:
Yeah, I was looking into parsing the year as an argument but that felt a little clunky, not to mention the fact that the user would also have to be aware that they are required to parse the arguments in order to run the program. This also raises the point of whether we should also parse the week number.

But yes I agree this could be discussed more at the meeting.